less/availity-angular.less

```diff
@@ -10,3 +10,17 @@
   background: none;
   z-index: 1011;
 }
+
+.dimmer-content {
```

Comment:
Fair enough. You could still inherit the same attributes from the overlay so they stay in sync.

If we don't inherit, can we still get a change for the width, height, and padding though? Padding should be relative to font size I would think rather than variable to the available width. Also, our other methods of "covering" a space is to do top, right, bottom, and left = 0. 

```
.card-img-overlay {
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  padding: 1.25rem;
}
```